zcash_test_vectors/orchard/asset_id.py

```diff
@@ -5,15 +5,28 @@
 
 import random
 
+from hashlib import blake2b
 from ..orchard.group_hash import group_hash
 from ..output import render_args, render_tv, option
 
 
 def native_asset():
     return group_hash(b"z.cash:Orchard-cv", b"v")
 
-def asset_id(key, description):
-    return group_hash(b"z.cash:Orchard-cv", key + description)
+
+def encode_asset_id(key, description):
+    return b"\x00" + key + description
+
+
+def asset_digest(encoded_asset_id):
+    digest = blake2b(person=b"ZSA-Asset-Digest")
+    digest.update(encoded_asset_id)
+    return digest.digest()
```

Comment:
I don't like `digest.digest()`. Let's rename `digest` to `h`.

Reply:
Done.